app/controllers/forms_controller.rb

```diff
@@ -41,8 +41,17 @@ def calculate_early_result
     if last_tag_in_group?(:gross_income)
       cfe_result = CfeService.result(session_data, Steps::Helper.completed_steps_for(session_data, step))
       session_data["early_result"] = { "result" => cfe_result.gross_income_result,
-                                       "gross_income_excess" => cfe_result.gross_income_excess }
-      # add analytics tracking here? completed user journeys
+                                       "gross_income_excess" => cfe_result.gross_income_excess,
+                                       "type" => "gross_income" } # I think we have to set this here so we have it in the session, when calling the `JourneyLoggerService`
+      track_completed_journey_for_early_result
```

Comment:
at the moment, this is also going to add an entry for an eligible result - we only want to track ineligible ones as a completed user journey. So we need to make sure we only save the ineligible ones.